lib/components/UploadList.jsx

```diff
@@ -69,13 +69,19 @@ var UploadList = React.createClass({
     var showDetailsThisUpload = _.includes(this.state.showErrorDetails, upload.key);
     var errorDetails = showDetailsThisUpload ? (<div className="UploadList-error-details">{upload.error.debug}</div>) : null;
     var showErrorsText = showDetailsThisUpload ? this.props.text.HIDE_ERROR : this.props.text.SHOW_ERROR;
-    
+    var errorMessage = upload.error.driverLink ? <div className="UploadList-error-message-wrapper">
```

Comment:
To see the error message with link to driver, just try to upload without a Dexcom connected. The driver works on pre-El Capitan as well, so there's no harm in Yosemite users installing it. In fact, if you have Portrait on the Web installed, you should already have it installed as well. To check, do a `kextstat | grep Dex` while the receiver is plugged in. If it displays something like `com.dexcom.driver.DexComUSB`, you've got it.